cmd/collector/app/metrics.go

```diff
@@ -96,19 +100,16 @@ func NewSpanProcessorMetrics(serviceMetrics metrics.Factory, hostMetrics metrics
 func newMetricsBySvc(factory metrics.Factory, category string) metricsBySvc {
 	return metricsBySvc{
 		spans: countsBySvc{
-			counts:  make(map[string]metrics.Counter),
-			factory: factory.Namespace("spans."+category, nil),
-			lock:    &sync.Mutex{},
-		},
-		debugSpans: countsBySvc{
-			counts:  make(map[string]metrics.Counter),
-			factory: factory.Namespace("debug-spans."+category, nil),
-			lock:    &sync.Mutex{},
+			counts:      make(map[string]metrics.Counter),
```

Comment:
I refactored this slightly. Rather than having debugSpans and debugTraces as separate metrics, they should be part of the spans and traces metrics but with a debug tag. This way it's easier to aggregate. I guess technically this is a breaking change?

While not in scope of this PR, we could break up the taggify the metrics further by splitting out service as a tag.